pom.xml

```diff
@@ -34,6 +34,11 @@
   <dependencies>
     <dependency>
       <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-sts</artifactId>
```

Comment:
This is something I'm working on to make it easier to launch MultiLang Daemon clients that might work for you.  It does require the addition of Maven, but only to create a launch script, and combined set of jars.

https://gist.github.com/pfifer/738c4f8b2eeb5757b1d8962d60e6f1e0

To build this you just run
```
mvn compile appassembler:assemble
```

That will create a launcher script, and jar directory in target/appassembler.  You just need to copy all the files in that to where ever you want the launcher to live, though the easiest is to co-locate it with where your properties file is located.

To run the application you need to do two steps:

1. Set the variable `CLASSPATH_PREFIX` to the directory where your properties file is located.
2. Start the daemon by running bin/multilang-daemon <properties files>
     * I've not tested this on Windows, but you should be able to run bin/multilang-daemon.bat

When using the generated script it will not use the jars download by the language specific module, but instead the jars that will be in the repo directory.

The example gist I've included adds 3 extra dependencies specifically logback, and slf4j bridges for Apache Commons Log.  
You should be able to add the STS credentials dependency, and then use the code in this PR to load, and use the new provider.

If you try this out please let me know how it works for you.  I haven't had much time to test this yet, but I do plan to do more testing.  This changes will make logging configuration for multilang daemon applications easier.